spec/serializers/provider_serializer_spec.rb

```diff
@@ -57,7 +57,25 @@
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
 
-  context "when UCAS preferences are missing" do
+  describe 'application alert recipient' do
+    context 'if not nil the application alert recipient is serialized upon instantiation of the contact object' do
```

Comment:
This doesn't feel like a context. Could we also reorganise this test to make it easier to follow?
```
context 'when set' do
  it 'does not serialise the application alert recipient' do
...
  end
end
```